core-web/src/main/resources/META-INF/viewGenerator.tld

```diff
@@ -1087,34 +1087,13 @@
       <rtexprvalue>false</rtexprvalue>
       <type>boolean</type>
     </attribute>
-    <attribute>
-      <description>URL to be used to export data.</description>
-      <name>exportURL</name>
-      <required>false</required>
-      <rtexprvalue>false</rtexprvalue>
-      <type>java.lang.String</type>
-    </attribute>
-    <attribute>
-      <description>Defines if lines can be explicitly sorted</description>
-      <name>sortableLines</name>
```

Comment:
This is yet used (favorite links for example)

Reply:
Yes only in two JSP/tag